app/src/main/java/org/simple/clinic/newentry/PatientEntryViewRenderer.kt

```diff
@@ -1,27 +1,36 @@
 package org.simple.clinic.newentry
 
 import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.patient.OngoingNewPatientEntry
+import org.simple.clinic.patient.businessid.Identifier
 import org.simple.clinic.util.DistinctValueCallback
 import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility
 import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility.AGE_VISIBLE
 import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility.BOTH_VISIBLE
 import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility.DATE_OF_BIRTH_VISIBLE
 
 class PatientEntryViewRenderer(val ui: PatientEntryUi) : ViewRenderer<PatientEntryModel> {
-  private val distinctDateOfBirthAndAgeVisibilityCallback: DistinctValueCallback<DateOfBirthAndAgeVisibility> = DistinctValueCallback()
+  private val distinctDateOfBirthAndAgeVisibilityCallback = DistinctValueCallback<DateOfBirthAndAgeVisibility>()
 
   override fun render(model: PatientEntryModel) {
     val patientEntry = model.patientEntry ?: return
+    showOrHideIdentifier(patientEntry.identifier)
 
-    if (patientEntry.identifier != null) {
+    val personalDetails = patientEntry.personalDetails ?: return
+    changeDateOfBirthAndAgeVisibility(personalDetails)
+  }
+
+  private fun showOrHideIdentifier(identifier: Identifier?) {
```

Comment:
`showOrHideIdentifier` leaves some ambiguity from the name. Should we just call it `renderIdentifier` instead? Whether to hide or do something else is an implementation detail.